app/js/components/Profile/UpdateUser.jsx

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react';
+import {Row, Col, Button} from 'reactstrap';
+import PropTypes from 'prop-types';
+import {withNamespaces} from 'react-i18next';
+
+class UpdateUser extends Component {
```

Comment:
That's a weird component. I feel like it is never used alone, but might be wrong.
Also, it could be stateless

Reply:
Yes Now that we have style for everything I will have a look to the components I can see this one being CenterButton or something similar, we have it in multiple place

Reply:
Yeah that makes more sense